core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -256,7 +226,7 @@ private[spark] object ClosureCleaner extends Logging {
       return
     }
 
-    if (lambdaFunc.isEmpty) {
+    if (maybeIndylambdaProxy.isEmpty) {
       logDebug(s"+++ Cleaning closure $func (${func.getClass.getName}) +++")
```

Comment:
nit: how about explicitly saying this is an old style closure here? e.g., `Cleaning old style closure $func`, `Cleaning inline closure $func`, ...?

Reply:
I left this part untouched in this PR intentionally, for the reason mentioned in a previous reply.

It's highly unlikely to actually see this log get triggered in Spark 3.0 in the real world, unless someone decided to go with `-Ydelambdafy:inline` explicitly w/ Scala 2.12+. So keeping it the way it is for Spark 3.0 wouldn't hurt much.

I do plan to send out a few cleanup/refactoring PRs to get the old code into a better shape, and maybe enhance the ClosureCleaner's capabilities, e.g. support cleaning "sibling" closures.